gdmix-workflow/images/gdmix_dev/Dockerfile

```diff
@@ -46,14 +45,14 @@ RUN rm -rf /usr/bin/python && \
 RUN sed -i 's/python/python2/g' /usr/bin/yum
```

Comment:
not related this PR, but why do we need to touch python2?

Reply:
Because the yum version in our image still needs python2

Reply:
Yeah, used ubuntu python 3.7 image as base image, no need to install python 3.7 any more.